rocky/assets/css/themes/soft/manon/kat-purrple.scss

```diff
@@ -1,12 +1,12 @@
 :root {
-    --kat-purrple-1: #F7F6FC;
-    --kat-purrple-2: #EEECF8;
-    --kat-purrple-3: #D6D1EF;
-    --kat-purrple-4: #BFB6E6;
-    --kat-purrple-5: #AF9BD3;
-    --kat-purrple-6: #8C7CD2;
-    --kat-purrple-7: #6E5FAE;
-    --kat-purrple-8: #5B4F90;
-    --kat-purrple-9: #4D437A;
-    --kat-purrple-10: #3D3560;
+  --kat-purrple-1: var(--colors-violet-1);
+  --kat-purrple-2: var(--colors-violet-2);
+  --kat-purrple-3: var(--colors-violet-4);
+  --kat-purrple-4: var(--colors-violet-4);
+  --kat-purrple-5: var(--colors-violet-7);
+  --kat-purrple-6: var(--colors-violet-8);
+  --kat-purrple-7: var(--colors-violet-9);
+  --kat-purrple-8: var(--colors-violet-10);
+  --kat-purrple-9: var(--colors-violet-11);
+  --kat-purrple-10: var(--colors-violet-12);
```

Comment:
I think `kat-purrple-4` should map to `colors-violet-5`, but I think it would be better to replace all usage of `kat-purrple` with `colors-violet` because having two definitions with different numbers is a bit confusing.